sycl/include/sycl/feature_test.hpp.in

```diff
@@ -62,7 +62,9 @@ __SYCL_INLINE_VER_NAMESPACE(_V1) {
 #define SYCL_EXT_INTEL_EXTENDED_ATOMICS 1
 #endif
 #endif
-#define SYCL_EXT_INTEL_FPGA_DEVICE_SELECTOR 1
+#ifndef SYCL_EXT_INTEL_FPGA_DEVICE_SELECTOR
```

Comment:
Is the #if needed?

Reply:
No, this is not right.  You should just have `#define SYCL_EXT_INTEL_FPGA_DEVICE_SELECTOR 2`

Reply:
removed